tests/gz-sim-yarp-plugins-check-model/CMakeLists.txt

```diff
@@ -17,6 +17,16 @@ gsyp_add_tutorial_model_check(forcetorque-model-one-sensor forcetorque/model_one
 gsyp_add_tutorial_model_check(forcetorque-model-two-sensors forcetorque/model_two_sensors/model2sensors.sdf)
 gsyp_add_tutorial_model_check(single-pendulum single_pendulum/model.sdf)
 
+if(ICUB_FOUND)
+    option(ENABLE_COUPLED_PENDULUM_TEST "Enable coupled pendulum tests that require ICUB" ON)
+else()
+    set(ENABLE_COUPLED_PENDULUM_TEST OFF)
+endif()
```

Comment:
I think this logic can be useful for all tests, so I think we can move it upper in the tests CMakeList.txt (see the comment there).